plugins/modules/gns3_project.py

```diff
@@ -284,6 +302,8 @@ def main():
             ),
             project_name=dict(type="str", default=None),
             project_id=dict(type="str", default=None),
+            scene_height=dict(type="int", default=3080),
```

Comment:
Do you happen to have/know the default values of the API for the `scene_height` and `scene_width`? Wondering if these are the values: https://gns3-server.readthedocs.io/en/stable/api/v2/controller/project/projects.html

I think is better to leave at the GNS3 API default values so it doesn't affect other users current functionality

Reply:
Hi, 

i think default values are 2000 and 1000. I'll double check this and will update the settings.

Reply:
Updated the code with GNS3 defaults.